### PR TITLE
Fixes related to missing external deps

### PR DESCRIPTION
**Pull request by dwlehman**
This is stuff I encountered while testing the API demo on a freshly-installed F26 Alpha Server installation.

### CODE REVIEW

**Comment by dwlehman** [comment]
Jenkins, test this please.

File: blivet/static_data/lvm_info.py [hunk 1]
```diff
@@ -37,7 +37,13 @@ class LVsInfo(object):
     @property
     def cache(self):
         if self._lvs_cache is None:
-            lvs = blockdev.lvm.lvs()
+            try:
+                lvs = blockdev.lvm.lvs()
+            except blockdev.BlockDevNotImplementedError:  # pylint: disable=catching-non-exception
```
Comment (vpodzime) [inline blivet/static_data/lvm_info.py]: It should be possible to catch ``NotImplementedError`` here too (``BlockDevNotImplementedError`` should be inheriting from it)

File: blivet/static_data/lvm_info.py [hunk 2]
```diff
@@ -56,8 +62,14 @@ class PVsInfo(object):
...
     def cache(self):
         if self._pvs_cache is None:
-            pvs = blockdev.lvm.pvs()
             self._pvs_cache = dict()  # pylint: disable=attribute-defined-outside-init
+
+            try:
+                pvs = blockdev.lvm.pvs()
+            except blockdev.BlockDevNotImplementedError:  # pylint: disable=catching-non-exception
+                log.error("libblockdev lvm plugin is missing")
+                return self._pvs_cache
```
Comment (vpodzime) [inline blivet/static_data/lvm_info.py]: Shouldn't this also initialize the cache to an empty dict if the plugin is missing like for ``lvs`` above? There's no point trying this multiple times (for every PV), right?

Comment (dwlehman) [inline blivet/static_data/lvm_info.py]: Yes, good catch.

Comment (dwlehman) [inline blivet/static_data/lvm_info.py]: Actually, it's not needed because the dict is initialized just above the changed code (within the same property).

Comment (vpodzime) [inline blivet/static_data/lvm_info.py]: I see it now. I got confused by two different ways the same is achieved for PVs and LVs.

**Pull request merged**